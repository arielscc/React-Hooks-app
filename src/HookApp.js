import React from 'react';

function HookApp() {
  return <div>Hola a todos</div>;
}

export default HookApp;
