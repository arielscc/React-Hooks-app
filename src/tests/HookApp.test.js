import React from 'react';

import { shallow } from 'enzyme';
import HookApp from '../HookApp';

import '@testing-library/jest-dom';
import toJson from 'enzyme-to-json';

describe('Probando el primer componente', () => {
  test('Debe hacer match con el snapshot', () => {
    const wrapper = shallow(<HookApp />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
