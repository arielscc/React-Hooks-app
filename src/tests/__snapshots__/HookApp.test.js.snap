// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando el primer componente Debe hacer match con el snapshot 1`] = `
<div>
  Hola a todos
</div>
`;
